### DISCUSSION TITLE
Is it possible to keep floating window always above?

### DISCUSSION THREAD

**Discussion by iacore** [qtile/qtile#3817] [General]
When I spawn a tiled window with `lazy.spawn`, it is spawned on top of floating windows.

I checked Issues, it seems to be a issue from long ago.
Related: https://github.com/qtile/qtile/issues/753 https://github.com/qtile/qtile/issues/1693

Is there a way to do that?

**Comment by iacore** [comment]
I did it!
```
from libqtile import hook

@hook.subscribe.group_window_add
def window_added(group, window):
    if window.floating:
        window.cmd_bring_to_front()
    else:
        for win in reversed(group.focus_history):
            if win.floating:
                win.cmd_bring_to_front()
                return
```

**Comment by elParaguayo** [comment]
There is an open PR which will add this functionality once merged.
